vault/resource_consul_secret_backend_test.go

```diff
@@ -211,13 +225,16 @@ resource "vault_consul_secret_backend" "test" {
 }`, path, token)
 }
 
-func testConsulSecretBackend_bootstrapConfig(path, addr string) string {
+func testConsulSecretBackend_bootstrapConfig(path, addr, token, bootstrap string) string {
```

Comment:
bootstrap is a boolean

```suggestion
func testConsulSecretBackend_bootstrapConfig(path, addr, token string, bootstrap bool) string {
```